src/utils/EquipmentType.ts

```diff
@@ -44,81 +44,117 @@ export const equipmentStyles = {
-    VOLTAGE_LEVEL: {
-        name: 'VOLTAGE_LEVEL',
+    [EquipmentType.VOLTAGE_LEVEL]: {
+        name: EquipmentType.VOLTAGE_LEVEL,
         tagLabel: 'equipment_search/voltageLevelTag',
     },
-    LINE: {
-        name: 'LINE',
+    [EquipmentType.LINE]: {
+        name: EquipmentType.LINE,
         tagLabel: 'equipment_search/lineTag',
     },
-    TWO_WINDINGS_TRANSFORMER: {
-        name: 'TWO_WINDINGS_TRANSFORMER',
+    [EquipmentType.TWO_WINDING_TRANSFORMER]: {
+        name: EquipmentType.TWO_WINDING_TRANSFORMER,
         tagLabel: 'equipment_search/2wtTag',
     },
-    THREE_WINDINGS_TRANSFORMER: {
-        name: 'THREE_WINDINGS_TRANSFORMER',
+    [EquipmentType.THREE_WINDINGS_TRANSFORMER]: {
+        name: EquipmentType.THREE_WINDINGS_TRANSFORMER,
         tagLabel: 'equipment_search/3wtTag',
     },
-    HVDC_LINE: {
-        name: 'HVDC_LINE',
+    [EquipmentType.HVDC_LINE]: {
+        name: EquipmentType.HVDC_LINE,
         tagLabel: 'equipment_search/hvdcLineTag',
     },
-    GENERATOR: {
-        name: 'GENERATOR',
+    [EquipmentType.GENERATOR]: {
+        name: EquipmentType.GENERATOR,
         tagLabel: 'equipment_search/generatorTag',
     },
-    BATTERY: {
-        name: 'BATTERY',
+    [EquipmentType.BATTERY]: {
+        name: EquipmentType.BATTERY,
         tagLabel: 'equipment_search/batteryTag',
     },
-    LOAD: {
-        name: 'LOAD',
+    [EquipmentType.LOAD]: {
+        name: EquipmentType.LOAD,
         tagLabel: 'equipment_search/loadTag',
     },
-    SHUNT_COMPENSATOR: {
-        name: 'SHUNT_COMPENSATOR',
+    [EquipmentType.SHUNT_COMPENSATOR]: {
+        name: EquipmentType.SHUNT_COMPENSATOR,
         tagLabel: 'equipment_search/shuntTag',
     },
-    DANGLING_LINE: {
-        name: 'DANGLING_LINE',
+    [EquipmentType.DANGLING_LINE]: {
+        name: EquipmentType.DANGLING_LINE,
         tagLabel: 'equipment_search/lineTag',
     },
-    STATIC_VAR_COMPENSATOR: {
-        name: 'STATIC_VAR_COMPENSATOR',
+    [EquipmentType.STATIC_VAR_COMPENSATOR]: {
+        name: EquipmentType.STATIC_VAR_COMPENSATOR,
         tagLabel: 'equipment_search/svcTag',
     },
-    HVDC_CONVERTER_STATION: {
-        name: 'HVDC_CONVERTER_STATION',
+    [EquipmentType.HVDC_CONVERTER_STATION]: {
+        name: EquipmentType.HVDC_CONVERTER_STATION,
         tagLabel: 'equipment_search/hvdcStationTag',
     },
-    BUSBAR_SECTION: {
-        name: 'BUSBAR_SECTION',
+    [EquipmentType.BUSBAR_SECTION]: {
+        name: EquipmentType.BUSBAR_SECTION,
         tagLabel: 'equipment_search/busbarSectionTag',
     },
-    BUS: {
-        name: 'BUS',
+    [EquipmentType.BUS]: {
+        name: EquipmentType.BUS,
         tagLabel: 'equipment_search/busTag',
     },
-    SWITCH: {
-        name: 'SWITCH',
+    [EquipmentType.SWITCH]: {
+        name: EquipmentType.SWITCH,
         tagLabel: 'equipment_search/switchTag',
     },
+    [EquipmentType.VSC_CONVERTER_STATION]: undefined,
+    [EquipmentType.LCC_CONVERTER_STATION]: undefined,
```

Comment:
We can't define them also ?

Reply:
I think they're both gathered under the HVDC_CONVERTER_STATION, but as I defined it as a Record<EquipmentType, > I had to define it with all the keys. I don't know where this object is used and not sure the tagLabel will make sense for them.